src/html/html-document.ts

```diff
@@ -63,6 +63,17 @@ export class ParsedHtmlDocument extends ParsedDocument<ASTNode, HtmlVisitor> {
         end: {line: location.endTag.line - 1, column: location.endTag.col - 1}
       };
     }
+    // text node value can provide accurate line and column count
```

Comment:
This comment wasn't helpful until after I'd already read and understood the code. Is this accurate?

`// We need to determine the ending line and column ourselves for other kinds of nodes. This is tricky and we probably don't do it right for all cases yet, but for text nodes at least we can just examine the node's content.`